data_handler/handler_tools/data_parser/serializers.py

```diff
@@ -86,3 +86,22 @@ def validate_valid_numbers(cls, value: str, info: ValidationInfo) -> Decimal:
         if not value.isdigit():
             raise ValueError("%s field is not numeric" % info.field_name)
         return Decimal(str(int(value, base=16)))
+
+class RepaymentEventData(BaseModel):
+    repayer: str
+    beneficiary: str
+    token: str
+    raw_amount: str
+    face_amount: str
+
+class RepaymentEventSerializer(BaseModel):
+    repayer: Any       # Replace `Any` with a more specific type if known (e.g., `str` or `int`)
```

Comment:
remove comment from CHAT GPT, these comments are useless and it's just recommendation.

Reply:
done